packages/docusaurus-plugin-content-docs/src/cli.ts

```diff
@@ -49,7 +49,8 @@ export function cliDocsVersionCommand(
 
   // Since we are going to create `version-${version}` folder, we need to make
   // sure it's a valid pathname.
-  if (/[<>:"\/\\|?*\x00-\x1F]/g.test(version)) {
+  // eslint-disable-next-line no-control-regex
+  if (/[<>:"|?*\x00-\x1F]/g.test(version)) {
```

Comment:
Removed check for `/` and `\`. It's already covered here so it's never reachable:

https://github.com/SamChou19815/docusaurus/blob/b0739a55da6dc8eedef7ecc172c55c1f88c474bd/packages/docusaurus-plugin-content-docs/src/cli.ts#L38-L42